src/Components/Sticky/Sticky.tsx

```diff
@@ -1,10 +1,10 @@
-import ReactSticky, { Props as ReactStickyProps } from "react-stickynode"
-import { __internal__useMatchMedia } from "Utils/Hooks/useMatchMedia"
 import { Box, themeProps } from "@artsy/palette"
-import { useEffect, useRef, useState } from "react"
 import * as React from "react"
-import { useSticky } from "./StickyProvider"
+import { useRef, useState } from "react"
```

Comment:
Not sure about this pattern because it's harder to read and write in my opinion. 

I usually use VSCode's auto import on save + automatic import folding to not see or care about imports anymore 🙃

Reply:
I agree that it is hard to read, so maybe we could update it to only import the needed from React?

Reply:
You're right, this mix isn't good. I've updated the file 👍